android/app/src/main/java/com/expensify/chat/MainApplication.java

```diff
@@ -55,14 +56,25 @@ protected JSIModulePackage getJSIModulePackage() {
         }
       };
 
+  // TODO: Use this to enable new architecture.
+  // private final ReactNativeHost mNewArchitectureNativeHost =
+	//       new MainApplicationReactNativeHost(this);
+
   @Override
   public ReactNativeHost getReactNativeHost() {
+    // if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
```

Comment:
Should uncomment this because it will end up being the same with the current BuildConfig anyways

Reply:
I wanted to avoid creating 2 ReactNativeHost instances by removing this https://github.com/Expensify/App/pull/9841/files#diff-bc1549da6355f7f82f324af9c667b78ff60672b6b80219062629fe4184e8fa93R60, then it won't compile if that doesn't exist.

Reply:
I could also just remove that code completely, but wanted to stay close to template

Reply:
Other option could be to do `private final ReactNativeHost mNewArchitectureNativeHost = null`